### ISSUE TITLE
linux-kernel: CVE-2017-12190

### ISSUE THREAD

**Issue by l2dy** [AOSC-Dev/aosc-os-abbs#815]
https://bugzilla.redhat.com/show_bug.cgi?id=1495089
http://www.openwall.com/lists/oss-security/2017/10/18/9

**Comment by MingcongBai** [comment]
@cthbleachbit Any updates?

**Comment by cthbleachbit** [comment]
I remember fixing it, check bugfix commits!

**Comment by MingcongBai** [comment]
@cthbleachbit Well try and close that with reference to commits next time...

**Comment by MingcongBai** [comment]
I don't think it's been fixed yet...

**Comment by cthbleachbit** [comment]
@MingcongBai Oh. I thought it's the SCSI problem

**Comment by MingcongBai** [comment]
@cthbleachbit This should be fixed no?

**Comment by liushuyu** [comment]
Current status for this one?

**Comment by l2dy** [comment]
Use AOSA-2017-0245 for the following CVEs fixed in aa74186c91207b3cf8211c57403bf7f05ba44f15.

```
CVE-2017-12188: Fixed with 4.13.8
CVE-2017-12190: Fixed with 4.13.8
CVE-2017-15265: Fixed with 4.13.8
CVE-2017-16525: Fixed with 4.13.8
CVE-2017-16527: Fixed with 4.13.8
CVE-2017-16533: Fixed with 4.13.8
```

Ref: https://github.com/nluedtke/linux_kernel_cves/blob/master/4.13/4.13_CVEs.txt
